terminate called after throwing an instance of 'std::out_of_range'
  what():  basic_string::erase
This is an example
is an example
i
bash: line 7: 30742 Aborted                 (core dumped) ./a.out
